cmd/cf-operator/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["main.go"],
+    importpath = "code.cloudfoundry.org/cf-operator/cmd/cf-operator",
+    visibility = ["//visibility:private"],
+    x_defs = {
+        "code.cloudfoundry.org/cf-operator/version.Version": "{STABLE_ARTIFACT_VERSION}",
+    },
+    deps = ["//cmd/internal:go_default_library"],
+)
+
+go_binary(
+    name = "cf-operator",
+    embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
+)
+
+go_image(
+    name = "image",
+    base = "@cf_operator_base//image",
```

Comment:
The base cf-operator image is referenced here and only the binary is appended as a layer on top of the image, optimizing the cache.